Source/Flow/Public/Interfaces/FlowContextPinSupplierInterface.h

```diff
@@ -0,0 +1,45 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#pragma once
+
+#include "UObject/Interface.h"
+#include "Nodes/FlowPin.h"
+
+#include "FlowContextPinSupplierInterface.generated.h"
+
+// A flow element (UFlowNode, UFlowNodeAddOn, etc.) that may supply context pins
+// "Context Pins" are those that can be dynamically added/removed to a FlowNode by property
+// settings on the flow node, by subobjects, etc.
+UINTERFACE(MinimalAPI, Blueprintable, DisplayName = "Flow ContextPin Supplier Interface")
+class UFlowContextPinSupplierInterface : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class FLOW_API IFlowContextPinSupplierInterface
```

Comment:
This is an interface for the ContextPin supplying functions (subgraph nodes, etc.)